_includes/cli.md

```diff
@@ -23,20 +50,60 @@
 
 ## Options
 
-| Name, shorthand | Default | Description |
-| ---- | ------- | ----------- |{% for option in  site.data[include.datafolder][include.datafile].options %}
-| `--{{ option.option }}{% if option.shorthand %}, -{{ option.shorthand }}{% endif %}` | {% if option.default_value and option.default_value != "[]" %}`{{ option.default_value }}`{% endif %} | {{ option.description | replace: "|","&#124;" | strip }} | {% endfor %}
-
-{% endif %}
+<table>
+<thead>
+  <tr>
+    <td>Name, shorthand</td>
+    <td>Default</td>
+    <td>Stability</td>
+    <td>Description</td>
+  </tr>
+</thead>
+<tbody>
+{% for option in site.data[include.datafolder][include.datafile].options %}
+  <tr>
+    <td markdown="span">`--{{ option.option }}{% if option.shorthand %} , -{{ option.shorthand }}{% endif %}`</td>
+    <td markdown="span">{% if option.default_value and option.default_value != "[]" %}`{{ option.default_value }}`{% endif %}</td>
+    <td markdown="span">
+    {% if option.deprecated == 'true' and option.experimental == 'true' %}
```

Comment:
If the YAML file is as such,

```yaml
deprecated: false
experimental: true
```

Those values would be treated as booleans but in the Liquid, aren't you checking them as strings? Either remove the quotes from `'true'` or just do `{% if option.deprecated and option.experimental %}` since they're booleans.